### ISSUE TITLE
Settings -> Option to Choose Open in New Instance or Open in New Tab

### ISSUE THREAD

**Issue by asjones987** [nomacs/nomacs#260]
This may actually already be there and I just can't find it (search website and settings).
Currently when I double click on an Image it opens a new instance of Nomacs.  So I have a large number of windows open when I might have preferred tabs. Is there or it would be nice to have a setting to default to open in tabs and not new windows/instances.

Seeing this on Windows 7 Nomacs 3.11

thanks

Alan

**Comment by diemmarkus** [comment]
Hi,

do you mean, when you double click in e.g. windows explorer? if so, it is related to #95 and not there yet - sorry!

best,
markus

**Comment by asjones987** [comment]
Yes, sorry I did not see bug #95.  Double clicking on any application. I had a coworker also send me screen shots in Skype and each one opened in a new Window.  In Some ways I see this bug related to #261 as if the option is set to open in an new tab I will want to "break" that to new windows an if I set it the other way I will want to combine windows to tabs :)

I am find with making this a dupe of #95.
